cli/domain/package-server-script.js

```diff
@@ -5,20 +5,41 @@ var format = require('stringformat');
 var fs = require('fs-extra');
 var path = require('path');
 var uglifyJs = require('uglify-js');
+var falafel = require('falafel');
 var _ = require('underscore');
 
 var hashBuilder = require('../../utils/hash-builder');
 var strings = require('../../resources');
 
+var wrapLoops = function(code){
+  var CONST_MAX_ITERATIONS = 1e9; // should this be configurable?
```

Comment:
Best would probably be to have a default (used during dev) and then get a configurable value on the registry level.

During publishing, CLI would ask the registry for packaging config and then use that value. 
But at the moment, I don't care, this is enough.
Can you just remove the comment and perhaps move the value here? https://github.com/opentable/oc/blob/master/resources/settings.js